### DISCUSSION TITLE
Custom HTTP status code in Error Handling

### DISCUSSION THREAD

**Discussion by saeed-rostami** [nuwave/lighthouse#2568] [General]
hello
I have a CustomException and use it like  
**throw new CustomException("Error Message" , 422);**

how can i modify the http status code for example 422 ?
it returns 200 always

```
<?php

namespace App\GraphQL;

use Exception;
use GraphQL\Error\ClientAware;
use GraphQL\Error\ProvidesExtensions;
class CustomException extends Exception implements ClientAware,ProvidesExtensions
{

    public function isClientSafe(): bool
    {
        return true;
    }


    public function getExtensions(): ?array
    {
        return [
            'reason' => $this->getMessage(),
        ];
    }

    /**
     * @return mixed
     */

}
```

**Comment by spawnia** [comment]
A similar question was asked in https://github.com/nuwave/lighthouse/issues/1694, and i believe the answer i gave with https://github.com/nuwave/lighthouse/issues/1694#issuecomment-766841328 also applies here.

**Comment by saeed-rostami** [comment]
ok thanks
im wonder if always status returnd 200  How should the client understand whether the data has arrived correctly or encountered an issue (validation error on auth error or ....)?
